typescriptSrc/interpreter.ts

```diff
@@ -282,6 +290,28 @@ module interpreter {
           }          
           stepper(vms, args);
         } 
+        else if (val instanceof ClosureV) {
+          const lambda = val.getLambdaNode();
+          const varStack = val.getContext();
+          const args = node.children(0, node.count());
+          const paramlist = lambda.child(0);
+          if(args.length != paramlist.children(0, paramlist.count()).length) {
+            vms.reportError("Number of arguments for lambda does not match parameter list.");
+            return
+          }
+          const manager = vms.getTransactionManager();
+          const stackFrame = new ObjectV(manager);
+          for (let i = 0; i < args.length; i++) {
+            const varName = paramlist.child(i).child(0).label().getVal();
+            const val = vms.getChildVal(i);
+            //TODO: check that the types of val and vardecl are the same
+            const field = new Field(varName, val, Type.ANY, false, false, manager);
```

Comment:
I think both flags should be true.  The constant flag should be true so that parameters can't be assigned. The declared flag should be true so that parameters can be accessed.